### ISSUE TITLE
[Bug]: Nothing happens on install

### ISSUE THREAD

**Issue by keytactic** [stephenjude/filament-jetstream#16]
### What happened?

After a fresh Laravel install, I installed this package with the --teams --api flag 
php artisan filament:jetstream:install --teams --api 

It installs livewire, filament, etc. But when I fire up the app, it looks like the standard Jetstream starter kit. 

I'm not sure what I'm missing. Are there additional steps required to achieve the jetstream/filament integration?

### How to reproduce the bug

Install Laravel (via Herd, no starter kit)
Install filament-jetstream package. 
Register a new user. 
Login. 

### Package Version

0.0.13

### PHP Version

8.2.0

### Laravel Version

11.0.0

### Which operating systems does with happen with?

Windows

**Comment by stephenjude** [comment]
Hi @keytactic 

Are you still having this issue? I am not able to reproduce this error:

![image](https://github.com/user-attachments/assets/e18c8711-e727-48f5-a0b8-225b626fee15)
![image](https://github.com/user-attachments/assets/bccefd8d-53ff-474e-b8c1-7117120cdb14)

**Comment by benbjurstrom** [comment]
Thought I hit the same bug last night, turns out the install process has an interactive menu about half way though. Just had to go back to the terminal and answer the question so the install could finish.

**Comment by stephenjude** [comment]
That could be it @benbjurstrom 

I am working on a v2 that won't require the whole installation of NPM

It's going to be the Filament way.

**Comment by frizcool** [comment]
I have the same issue. After installation, I ended up with two login routes: one for Filament at /admin/login and another for Jetstream at /app/login. When I try to merge the routes, nothing changes. I am using Filament v3.

**Comment by keytactic** [comment]
> Thought I hit the same bug last night, turns out the install process has an interactive menu about half way though. Just had to go back to the terminal and answer the question so the install could finish.

Yep. I did not realize this and that's exactly what it was. Using this on a new project and it's working wonderfully. 

Sorry about the false flag @stephenjude!

**Comment by stephenjude** [comment]
Thanks @keytactic 

I am also working on an improved version of this package that will reduce all these frictions greatly
